src/acm.cpp

```diff
@@ -699,6 +703,11 @@ bool ASN1_Codec::make_loggers( bool remove_files ) {
     return true;
 }
 
+bool ASN1_Codec::make_loggers_testing() {
+    logger = std::make_shared<AcmLogger>("testlog");
```

Comment:
Can we use make_unique instead? What is this hardcoded testlog string? Can we make the string a constant with a more descriptive name? Can we rename this function name to be more specific? It sounds like it makes loggers for testing? Maybe rename it to set_logger_name or something that tell what the function is about?

Reply:
Apparently 'make_unique' is only available from C++14 onwards.

The logger setup code has been refactored to improve clarity.